### ISSUE TITLE
Transfer-Encoding: chunked with proxy enabled

### ISSUE THREAD

**Issue by migliori** [serp-spider/search-engine-google#112]
Hi,

I've been using search-engine-google for a while, worked perfectly until now but I've got a recent issue with proxies.

The Google scraper works fine on my localhost, but on the production server it throws an error 500: Unable to check javascript status

The scraped results come with dom => textContent starting with "ncoding
Transfer-Encoding: chunked"

I put a simple test online here: https://www.hack-hunt.com/scraping-simple-test.php
The code is the code of your example here: http://serp-spider.github.io/documentation/search-engine/google/#installation

I just added:
$proxy = Proxy::createFromString('https://xxx:proxy@ip');
$browser->setProxy($proxy);

It works fine on localhost, or on production server if I remove the proxy, but it fails on production with proxy.

Not sure if the issue comes from my server or search-engine-google.

Any help much appreciated, thanks

**Comment by LunarDevelopment** [comment]
Can you post the contents of your local composer.json and production
composer.json & composer.lock  ?

**Comment by migliori** [comment]
[composer.zip](https://github.com/serp-spider/search-engine-google/files/2449573/composer.zip)

same on local & server

**Comment by LunarDevelopment** [comment]
Hm, looks alright - can you check that your production server IP is
whitelisted / isn't blocked with the proxy service ?

**Comment by gsouf** [comment]
@migliori Please check your CURL version. If curl version is not the same on the server, please try to upgrade and let us know what's going on.

**Comment by migliori** [comment]
No, it isn't, if you open [https://www.hack-hunt.com/scraping-simple-test.php](url) you'll see the string added before Google content:
`public 'textContent' => string 'ncoding
Transfer-Encoding: chunked

simpsons - Recherche Google(function(){window.google=...`

I suspected that headers could be added by Apache pagespeed module, but tried to disable it without success.

I can't change my PHP Curl version, it's built-in with PLESK PHP.
version =>7.26.0
ssl_version => OpenSSL/1.0.1t
libz_version => 1.2.7

I just tested with nginx instead of apache: same result.

**Comment by gsouf** [comment]
@migliori not php-curl, just curl itself. Run ``curl --version``

**Comment by gsouf** [comment]
You very of curl is very old. Try to upgrade to version 7.61 and see if it works.

Additionally curl <7.48 has issue with cookies, preventing SERPS to work correctly with cookies.

**Comment by migliori** [comment]
I'm in touch with my server provider & let you know if it's ok or not as soon as the upgrade is done - may take 1 or 2 days.

Thanks so much for your reactivity & help
